Profile-ModelarDB-Compression/main.py

```diff
@@ -0,0 +1,118 @@
+""" A script that computes which part of modelardbd's compressed storage format
```

Comment:
Maybe move the text down and align it all the way to the left as per the [conventions](https://peps.python.org/pep-0257/) for multi-line docstrings.

Reply:
I decide to deviate slightly from PEP 257 by creating a two line summary line as I did not have anything more to add to the docstring and I think it makes it harder to read by splitting.